tasks/elasticsearch-security.yml

```diff
@@ -17,27 +17,27 @@
   tags:
     - certificates
 
-- name: create keystore
+- name: Create keystore
   command: /usr/share/elasticsearch/bin/elasticsearch-keystore create
   args:
     creates: /etc/elasticsearch/elasticsearch.keystore
 
-- name: check for bootstrap password
+- name: Check for bootstrap password
   command: /usr/share/elasticsearch/bin/elasticsearch-keystore list
   changed_when: false
   register: es_keystore
 
-- name: set bootstrap password
+- name: Cet bootstrap password
   shell: >
     echo "{{ elasticsearch_bootstrap_pw }}" |
     /usr/share/elasticsearch/bin/elasticsearch-keystore
     add -x 'bootstrap.password'
   when: "'bootstrap.password' not in es_keystore.stdout_lines"
   ignore_errors: "{{ ansible_check_mode }}"
```

Comment:
I would put **change_when: false**

Reply:
Ahhh you put the password, you do not get it, is this right?

Reply:
Yes, in this task I set it. This change only changes the name of the task (lower case start of a task name is now considered lint)